.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -41,6 +41,7 @@ body:
         - Android
         - Windows
         - Other (specify in Anything else section)
+        - Not Applicable
```

Comment:
Just for context, what is the use case for this?

Reply:
for example this issue https://github.com/input-output-hk/catalyst-ci/issues/103 does not related to some specific OS on which user can run it.
I assume mainly it should be applicable to bugs with CI/CD.